packages/@lwc/engine-dom/src/renderer.ts

```diff
@@ -17,6 +17,12 @@ import {
 import { Renderer } from '@lwc/engine-core';
 
 const globalStylesheets: { [content: string]: true } = create(null);
+
+if (process.env.NODE_ENV === 'development') {
+    // @ts-ignore
+    window.__lwcGlobalStylesheets = globalStylesheets;
+}
+
```

Comment:
If there is a cleaner way to give access to this object from the Karma tests, let me know. Seems pretty low-risk to expose this in dev mode.

Reply:
Instead of exposing the `globalStylesheet` object directly on window, I would recommend exposing something like a `__lwcResetGlobalStyleSheets` function that would in charge of flushing out the object.